### DISCUSSION TITLE
Government employees can't access my website

### DISCUSSION THREAD

**Discussion by timathomas** [community/community#37987] [Other Feature Feedback, Questions, & Ideas]
Hi! My website evictionresearch.net is a tool to help folks see where evictions have been happening. Staff from several U.S. governmental agencies including the VA, White House, and HUD have told me that they are unable to access my website from their office computers. I'm assuming they may need some kind of certificate from my website. If that's the case, how do I add one? If that's not the case, what can I do to allow my page to be accessible to them? 

Here's the public repo https://github.com/evictionresearch/evictionresearch.github.io

**Comment by yoannchaudet** [comment]
Hello!

Without further details from them, we are not going to be able to help much I am afraid.

Your site is published at https://evictionresearch.net and has a valid certificate. What kind of error are they seeing?

**Comment by david-engelmann** [comment]
We're they trying to go through the evictions.study url?

**Comment by timathomas** [comment]
I believe they were using evictionresearch.net. One of the folks at the VA got `VA Security Policy Violation - Web Application Blocked (Restricted URL) (evictionresearch.net)` and their IT is saying that the webpage has to be approved. I'm not sure if other fed agencies got this block but I assume so. I can't imagine that all new websites need to be approved! I asked them directly what might be causing this so hopefully they'll provide an answer. I'll update if I find out anything.

**Reply by yoannchaudet** [reply]
"Web Application Blocked" does sound like they have a firewall. Let us know when you hear more!

**Comment by timathomas** [comment]
Here's the explanation from the VA: 

> “Network Edge Operations has received your request to gain access to https://evictionresearch.net. Appropriate actions have been taken through ***the vendor to have this site recategorized from parked, which is blocked by the VA, to the reference-and-research which is allowed***. NEO has verified this website now works on the VA network, and will be closing this ticket.

Now to figure out how to recategorize from parked to reference-and-research. Any tips would be appreciated!

**Comment by jfullstackdev** [comment]
can you use VPN ? so they can access your site ? even China can access GitHub using VPN, as others say

**Reply by timathomas** [reply]
I just checked the site and it's recategorized as "education." I think it took a bit for bots to scrape and categorize it, I'll see if my colleagues can access it or not.
